### ISSUE TITLE
Regression: The inserter search results don't use an audible message any longer

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#10583]
Previously, when searching for a block type in the inserter, the number of search results (or "no results") was made available to users of assistive technologies by sending a message to the assertive live region. The screenshot below is from Gutenberg 2.5:

![screen shot 2018-10-13 at 19 48 14](https://user-images.githubusercontent.com/1682452/46908323-8d856480-cf21-11e8-8e66-3754e4463d92.png)

Since then, the inserter has gone through various iterations and is now notably different. However, the several changes to the UI have apparently missed to re-implement the audible messages. On current master 4.0.0-rc.1 the search results are not announced at all, no message is sent to the live region.

This is a pretty important regression, as now users don't have a clue what happens when they type something in the search field.

**Comment by LukePettway** [comment]
So far from what I can see `debounceSpeak` doesn't seem to be called anywhere within any of the components callbacks. Looking at other components such as `hierarchical-term-selector`, it looks like it _should_ be explicitly calling that method. 

I'll play around with this more but I think I may be close to a solution.

**Comment by youknowriad** [comment]
Looks like this is fixed by #10755
